modules/wordads/wordads.php

```diff
@@ -87,12 +107,15 @@ function init() {
 		require_once WORDADS_ROOT . '/php/params.php';
 		$this->params = new WordAds_Params();
 
-		if ( is_admin() ) {
-			require_once WORDADS_ROOT . '/php/admin.php';
+		if ( $this->should_bail() ) {
 			return;
 		}
 
-		if ( $this->should_bail() ) {
+		require_once JETPACK__PLUGIN_DIR . '/extensions/blocks/wordads/wordads.php';
```

Comment:
Given that the file will be required anyway (all the block files in that directory are), I wonder if we should remove this and the registration below, and instead do it all in `/extensions/blocks/wordads/wordads.php`, just like the Markdown block:

```php
if (
	( defined( 'IS_WPCOM' ) && IS_WPCOM )
	|| ( method_exists( 'Jetpack', 'is_module_active' ) && Jetpack::is_module_active( 'wordads' ) )
) {
```

Reply:
It's in MU plugins, will the code be included automatically? I don't know much about how these specific files are getting included generally or if they're all expected to be manually included.

Reply:
D24713-code